src/main/java/ch/securify/decompiler/evm/OpCodes.java

```diff
@@ -286,6 +293,9 @@ public static int getPopCount(int opcode) {
 			case CALLCODE: return 7;
 			case RETURN: return 2;
 			case DELEGATECALL: return 6;
+			case STATICCALL: return 6;
+			case REVERT: return 2;
+			case INVALID: return 0;
```

Comment:
It is not defined in the yellow paper (`∅`), not sure whether we need to give this arbitrary value instead, at least worth a comment.